unimport/session.py

```diff
@@ -26,7 +27,9 @@ def _read(self, path):
         else:
             return source, encoding
 
-    def _list_paths(self, start, pattern="**/*.py"):
+    def _list_paths(
+        self, start: Path, pattern: str = "**/*.py"
+    ) -> Iterator[Path]:
         start = Path(start)
```

Comment:
If `start` is already a path, can we skip this step?

Reply:
> If `start` is already a path, can we skip this step?

yes it is already a path, https://github.com/hakancelik96/unimport/blob/d745cb33a37a27ba6715dc35eaad231a3621472f/unimport/__main__.py#L16